receiver/kafkareceiver/factory.go

```diff
@@ -40,6 +40,13 @@ const (
 	defaultAutoCommitEnable = true
 	// default from sarama.NewConfig()
 	defaultAutoCommitInterval = 1 * time.Second
+
+	// default from sarama.NewConfig()
+	defaultMinFetchSize = 1
```

Comment:
I'm wondering if we want to consider allowing these to be configured by MB instead of by bytes. 

I went with bytes because that's what samara is doing, but could easy just do some quick math to convert MB to bytes, open to thoughts on that. Seems more user-friendly.

Reply:
That said, minFetchSize is 1 byte, so not sure how that would look.

Reply:
IIRC kafka cluster settings and tunables that relate to fetching data from partitions are all configured in bytes so I think bytes if the correct choice.

Reply:
Yeah I decided against it after I realized minFetchSize was 1 byte by default and the implications of setting it to unlimited/0